src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts

```diff
@@ -341,6 +341,7 @@ export class AddFilterWizardComponent implements OnInit {
       this.validChildConfigurationForm = true;
       this.configurationData = null;
       this.pluginConfiguration = null;
+      pluginInfo.config = this.configurationControlService.getValidConfig(pluginInfo.config);
```

Comment:
among all `this.` move it to the top? and an inline comment explaining the case.